recipes/irfinder/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: irfinder
+  version: '1.2.2'
+  
+build:
+  number: 0
+  
+source:
+  git_rev: v1.2.2
+  url: https://github.com/williamritchie/IRFinder/archive/v1.2.2.tar.gz
+  md5: bf5d672b7ff61feaf24d1299b87fa2f1
+  
+requirements:
+  build:
+    - gcc
+  run:
+    - perl
+    - r-base
+      
+test:
+  commands:
+    - 'IRFinder -v &> /dev/null'
```

Comment:
Mmm, after checking actually I'm not sure what could be added here. Few information is available in the manual and article about the other scripts present in bin/. I don't see here what informative test could be done which would not involve data test (which we are not supposed to insert here right ?).